vignettes/GGIR.Rmd

```diff
@@ -609,23 +922,31 @@ if(length(args) > 0) {
 GGIR(f0=f0,f1=f1,...)
 ```
 
-You will need to update the `...` in the last line with the arguments you used for `GGIR`. Note that `f0=f0,f1=f1` is essential for this to work. The values of `f0` and `f1` are passed on from the bash script.
+You will need to update the `...` in the last line with the arguments
+you used for `GGIR`. Note that `f0=f0,f1=f1` is essential for this to
+work. The values of `f0` and `f1` are passed on from the bash script.
 
-Once this is all setup you will need to call `bash submit.sh`
-from the command line.
+Once this is all setup you will need to call `bash submit.sh` from the
+command line.
 
 **Important Note:**
 
-Please make sure that you process one GGIR part at the same time on a cluster, because each part assumes that preceding parts have been ran. You can make sure of this by always specifying argument `mode` to a single part of GGIR. Once the analysis stops update argument `mode` to the next part until all parts are done. The speed of the parallel processing is obviously dependent on the capacity of your computing cluster and the size of your dataset.
-
+Please make sure that you process one GGIR part at the same time on a
+cluster, because each part assumes that preceding parts have been ran.
+You can make sure of this by always specifying argument `mode` to a
+single part of GGIR. Once the analysis stops update argument `mode` to
+the next part until all parts are done. The speed of the parallel
+processing is obviously dependent on the capacity of your computing
+cluster and the size of your dataset.
```

Comment:
Remove these sentences, because this is no longer needed and I feel this is the responsibility of the researcher to think about. We cannot provide detailed guidance on how they should use a cluster.

Reply:
Agree and done.